README.md

```diff
@@ -23,29 +23,39 @@ auspice -help
 ```
 
 ## Run the build
-The build can be run at once or by workflows. Running the build by workflows can be helpful when troubleshooting or when testing modifications.
+This build can process and output global or Washington state focused WNV information. The build can also be run by workflows which is helpful when troubleshoting or all at once.
+
 To run the build by workflows first run the ingest workflow
 ```bash
 nextstrain build ingest
 ```
 Inside the ingest folder there should be two output files: metadata_all.tsv and sequences_all.tsv
 
 Run the phylogenetic workflow
+Execute the global build
 ```bash
 nextstrain build phylogenetic
 ```
+Or execute the Washington focused build 
+```bash
+nextstrain build phylogenetic --configfile build-configs/washington-state/config.yaml
+```
 Inside the phylogenetic folder there should be at least one output file: WNV-nextstrain_NA.json
 
-Run the build all at once
+Run the build all at once. This option defaults to the global build.
 ```bash
 nextstrain build
```

Comment:
Update instruction to run: nextstrain build phylogenetic

Reply:
Resolved with https://github.com/nextstrain/WNV/pull/51/commits/7ea2f9ad639a4e34a7cfb3552f721d336f992a4c